src/lib/salad/rtree.c

```diff
@@ -874,20 +972,21 @@ rtree_iterator_process_neigh(struct rtree_iterator *itr,
 		struct rtree_page_branch *b;
 		b = rtree_branch_get(itr->tree, pg, i);
 		coord_t distance;
-		if (itr->tree->distance_type == RTREE_EUCLID)
-			distance = rtree_rect_neigh_distance2(&b->rect,
-							      &itr->rect, d);
-		else
-			distance = rtree_rect_neigh_distance(&b->rect,
-							     &itr->rect, d);
+		if (itr->tree->distance_type == RTREE_EUCLID) {
+			distance = rtree_rect_neigh_distance2(
+				&b->rect, &itr->rect, d);
+		} else { /* RTREE_MANHATTAN */
+			distance = rtree_rect_neigh_distance(
+				&b->rect, &itr->rect, d);
+		}
 		struct rtree_neighbor *neigh =
-			rtree_iterator_new_neighbor(itr, b->data.page,
-						    distance, level - 1);
+			rtree_iterator_new_neighbor(
+				itr, b->data.page, distance,
+				level - 1);
```

Comment:
Again, you don't need these changes - they are purely cosmetic. Let's remove them.